index.rst

```diff
@@ -153,13 +186,13 @@ to return diagnostic information when something goes wrong with your
 application.  You'll see it in the example below.
 
 Example APIKit usage
---------------------
+^^^^^^^^^^^^^^^^^^^^
 
 The following describes how you would use :py:class:`apikit.APIFlask` to
 create a service wrapper suitable for use on ``api.lsst.codes``.
 
 Microservice server
-^^^^^^^^^^^^^^^^^^^
+@@@@@@@@@@@@@@@@@@@
```

Comment:
Suggest `"` for this level of headings — https://developer.lsst.io/docs/rst_styleguide.html#sections